share/wake/lib/system/remote_cache_api.wake

```diff
@@ -705,11 +693,13 @@ def timeoutErrorMessage: String =
 def cascadeTimeoutErrorMessage: String =
     "rsc: Remote cache previously disabled due to timeout"
 
-def randomUUID Unit: Result String Error =
+# Generates a UUID (using date in nano seconds) for the current wake invocation.
```

Comment:
Hurrah for portability!  Slight nitpick that this makes it not a UUID, but just a reasonably-unique string.  (It's kind of impressive how much design work has gone into avoiding accidental collisions for true UUIDs.)